lib/liquid/standardfilters.rb

```diff
@@ -306,6 +311,11 @@ def remove_first(input, string)
       input.to_s.sub(string.to_s, '')
     end
 
+    # remove the last occurences of a substring
+    def remove_last(input, string)
+      input.to_s.reverse.sub(string.to_s.reverse, '').reverse
```

Comment:
This seems quite inefficient.  Couldn't this be done more efficiently with [String#rindex](https://ruby-doc.org/core-3.0.2/String.html#method-i-rindex) to search for the substring to remove?

It also looks like `remove_last` could delegate to `replace_last` with an empty replacement string.  Seems like that simplification could be made with `remove_first` and `replace_first`.

Reply:
I have changed it to using rindex. This should be faster, especially on large inputs. It got a little more complicated, but I think it's fine. 

I also delegated all `remove` to `replace` with a third argument in case the default gets empty string gets removed.

Reply:
Updated to using Regex. It seems like a better option.